build/simp-gpgkeys.spec

```diff
@@ -94,6 +94,12 @@ chown -R root:48 ${dir}/GPGKEYS/
 find ${dir}/GPGKEYS/ -type f -exec chmod 640 {} +
 
 %changelog
+* Mon Sep 17 2018 Brandon Riden <brandon.riden@onyxpoint.com> - 3.0.4-0
+- Update SIMP-6 Release Key to include new signatures
+
+* Fri Jun 29 2018  Liz Nemsick <lnemsick.simp@gmail.com> - 3.0.4-0
```

Comment:
this doesn't have to be noted, as it is an internal change that only affects our release process.